web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +63,55 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
+    it("displays when the For You section is empty", () => {
+      render(
+        <SidebarCardsList
+          topCards={[]}
+          bottomCards={[]}
+          fundings={[]}
+          hiddenFundings={[]}
+          certifications={[]}
+          hiddenCertifications={[]}
+          displayFundings={false}
+          displayCertifications={false}
+        />
+      );
+
+      expect(screen.getByTestId("empty-for-you-message")).toBeInTheDocument();
```

Comment:
[pebble] We can get this item using `getByText()` and referencing the `config`. We can also drop the `test-id` on the component.